### ISSUE TITLE
New Rule Proposal: SA1404 but for ExcludeFromCoverage

### ISSUE THREAD

**Issue by mu88** [DotNetAnalyzers/StyleCopAnalyzers#3877]
I'd appreciate having a warning for cases when `[ExcludeFromCoverage]` is used without a justification, similar to `SA1404`.

For example, the following code would be a rule violation...

```csharp
[ExcludeFromCoverage]
public bool Enable()
{
}
```

...and adding a justification would fix the violation:

```csharp
[ExcludeFromCoverage(Justification = "Cannot be tested because...")]
public bool Enable()
{
}
```

If you consider this useful, I'd happily provide a PR.
